client/src/App.js

```diff
@@ -8,6 +8,7 @@ import {
 } from '@material-ui/core/styles';
 
 import theme from './component/theme/theme';
+import CartCard from './component/card/index';
```

Comment:
please don't add your component to app.js, and don't forget to create a new branch after checkout out to main when start working on a new task

Reply:
done :+1: